odoo/models.py

```diff
@@ -2045,7 +2045,7 @@ def _read_group_prepare(self, orderby, aggregated_fields, annotated_groupbys, qu
         :return: (groupby_terms, orderby_terms)
         """
         orderby_terms = []
-        groupby_terms = [gb['qualified_field'] for gb in annotated_groupbys]
+        groupby_terms = [gb['groupby'] for gb in annotated_groupbys]
```

Comment:
It's a temporary change. It should be something like

```suggestion
        groupby_terms = [gb['compute_raw'] or gb['qualified_field'] for gb in annotated_groupbys]
```